Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org' do
   gem 'inspec', '4.18.100'
   gem 'inspec-bin', '4.18.100'
-  gem 'kramdown', '2.1.0'
+  gem 'kramdown', '2.3.1'
```

Comment:
I have been getting a security warning on this so updating it as part of this change.